packages/loopring_v3/contracts/amm/LoopringAmmPool.sol

```diff
@@ -54,6 +71,17 @@ contract LoopringAmmPool is
         _;
     }
 
+    constructor(
+        IAmmController _controller,
```

Comment:
add set methods for constructor parameters?

Reply:
These addresses will set immutable variables in the contract, so they will be hardcoded in the implementation contract itself, not in storage. All pools (especially now) will use the same `controller` and `assetManager`, so I don't really see any reason to make these parameters configurable or updateable. By hard-coding these addresses in the implementation contract they are automatically used in all pools. By making the `controller` and `assetManager` contract updateable as well we can update the logic in these contracts by updating a single contract (instead of having to update every pool contract independently).

Reply:
OK, got it.